app/static/playlist.style.css

```diff
@@ -118,6 +119,46 @@ p:last-child {
   color: #fff;
 }
 
+/* COLLECT - BOTTOM RIGHT */
+
+.collect.collect-right:before {
```

Comment:
Perhaps rename `collect-right` to just `right`?

Reply:
Although this project is too small for class name management to be an issue, I still like to do it to some degree on principle. In this case I'm happy to make it 'right' because the other .js/py code will benefit for sure. Seems worth it.

Side note: For new projects going forward I'd like to adopt some sort of standard, such as [BEM](http://getbem.com/naming/). Of course it wouldn't make sense to use it here, but I'd be interested to know your thoughts.

Reply:
Yes for sure to BEM! :-)